main2.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, render_template, request, redirect, url_for, session
+
+app = Flask(__name__)
+
+
+app.secret_key = "rrgtqeyq43t43t45t4qter"
+
+@app.get('/submit')
+def submit_get():
+    context = {
+        'login': 'Авторизация'
+    }
+    return render_template('form.html', **context)
+
+
+@app.post('/submit')
+def submit_post():
+    session['login'] = request.form.get('login')
+    session['email'] = request.form.get('email')
+    return redirect(url_for('success'))
+
+
+@app.route('/success/', methods=['GET', 'POST'])
+def success():
+    if 'login' in session:
+        context = {
+            'login': session['login'],
+            'email': session['email'],
+            'title': 'Добро пожаловать'
+        }
+        if request.method == 'POST':
+            session.pop('login', None)
+            session.pop('email', None)
+            return redirect(url_for('submit_get'))
```

Comment:
Тут можно чуть иначе, с импортом соответствующих объектов
```suggestion
            response = make_response(redirect(url_for('submit_get')))
            response.delete_cookie('login',)
            response.delete_cookie('email')
            return response
```